src/uu/mktemp/src/mktemp.rs

```diff
@@ -124,9 +125,8 @@ struct Options {
 
     /// The directory in which to create the temporary file.
     ///
-    /// If `None`, the file will be created in the current directory.
+    /// If `None`, the file will be created in the OS-temp directory.
```

Comment:
I don't think this updated comment is correct, even with your other changes. It will only be None, when no `--tmpdir` or `-p` argument is given, but a template is given, in which case the file should be created in the current directory (unless the template is itself absolute).